.github/workflows/add-artifacts-to-release.yml

```diff
@@ -24,10 +24,11 @@ jobs:
         with:
           ref: ${{ steps.set_environment.outputs.git_ref }}
           fetch-depth: 0
-      - name: Set up JDK 1.11
-        uses: actions/setup-java@v1
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
         with:
-          java-version: 1.11
+          distribution: 'zulu'
```

Comment:
Should here and our docker files use the same distribution?

Reply:
Not necessarily. In fact, it might be even useful to check different distributions as this might help catch some incompatibilities which we would like to avoid.